src/app/compatibility/avm2/tree.svg/route.tsx

```diff
@@ -29,6 +29,9 @@ class Id {
 
 export async function GET() {
   const { document } = new JSDOM().window;
+  const releases = await getLatestReleases();
+  const latest = releases.length > 0 ? releases[0] : null;
+  const report_data = await fetchReport(latest?.avm2_report_asset_id || 150005259);
```

Comment:
Rather than this, we should probably use the most recent report if a Nightly is missing a report. Not sure the best code for that.

Reply:
Done